NSS-Day-Cohort-42/rare-api-fakes-news
# Cw comments

#113 Viewing Posts comments..
​

Description
Can View comments if attached to post.. and they are author.. this is just the serverside code for it..
​

How do I access this branch?
git fetch --all
2.git pull origin cw-comments​
Motivation and Context
​

How Has This Been Tested?
1.Make sure you have both branches!! clientside and serverside
2. Make sure you have comments on posts to check
3.Every post should have a comments button even if they are not an author or admin
4. The comments should show up on posts that have any
5. check the back to posts button works..
​

Screenshots (if appropriate):
​

Types of changes
 Bug fix (non-breaking change which fixes an issue)
[ x] New feature (non-breaking change which adds functionality)
 Breaking change (fix or feature that would cause existing functionality to not work as expected)
​
Checklist:
 My code follows the code style of this project.
 My change requires a change to the documentation.
 I have updated the documentation accordingly.
